### PR TITLE
[PVR] Krypton: Quick and dirty fix for trac #17374.

### PR DESCRIPTION
**Pull request by ksooo**
This a quick and dirty fix for http://trac.kodi.tv/ticket/17374 
Proper fix for Leia will be provided by #11905

The original patch was supplied and tested by the trac ticket submitter.

@Jalle19 mind taking a look.

### CODE REVIEW

**Comment by ksooo** [comment]
jenkins build this please

File: xbmc/pvr/PVRManager.cpp [hunk 1]
```diff
@@ -493,8 +493,11 @@ void CPVRManager::Unload()
 
 void CPVRManager::Shutdown()
 {
+// quick and dirty Krypton-only fix for http://trac.kodi.tv/ticket/17374
+#if 0
   // set system wakeup data
   SetWakeupCommand();
```
Comment (ksooo) [inline xbmc/pvr/PVRManager.cpp]: Why? I just was in the mood. And as this is krypton-only quick and dirty I don't really care, tbh. I don't think I will change this. I don't want to spend more time on this.

**Review by Jalle19 — APPROVED** [review]
I would also just remove the lines instead of using preprocessor directives, but it's not a big deal

**Comment by ksooo** [comment]
jenkins build and merge

@Jalle19 @xhaggi I removed the <code>#if 0</code>

**Comment by xhaggi** [comment]
Thanks

**Comment by ksooo** [comment]
jenkins build this please

**Pull request merged**